cli/src/main.rs

```diff
@@ -257,7 +258,11 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
             // write to file
             if ADDRESS_REGEX.is_match(&cmd.target).unwrap() {
-                output_path.push_str(&format!("/{}/snapshot.csv", &cmd.target));
+                output_path.push_str(&format!(
+                    "/{}/{}/snapshot.csv",
+                    chain_id(&cmd.rpc_url).await.unwrap(),
```

Comment:
this looks great, can we also implement this for the other modules?:
- decompile
- cfg
- disassemble

Reply:
sure thing

done :wink: